docs/README.md

```
@@ -0,0 +1,176 @@
+
+# Building Static HTML Pages
+
+**In most cases is unnecessary**. Running the `development server` will be sufficient.
+
+If you need to generate static HTML pages (unlikely), then follow these steps.
+
+```console
+$ yarn run build
+```
+
+This command generates static content into the `/build` directory and can be served using any static contents hosting service.
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)